### PR TITLE
TRW-1407 - allowed popular dimensions as prop to universal search

### PR DESCRIPTION
**Pull request by troyblank**
The ability to add categories to autocomplete has been done in this past. This is to add this prop to universal search.

### CODE REVIEW

File: components/universalSearch/searchBar.js [hunk 1]
```diff
@@ -83,11 +83,13 @@ export default class SearchBar extends PureComponent {
...
     render() {
         const { dimension, value, dimensionInputFocused, queries } = this.state;
-        const { dimensions } = this.props;
+        const { popularDimensions, dimensions } = this.props;
         const dimensionPickerButtonText = dimension || 'Choose a dimension...';
         const hasQueries = Boolean(queries.length);
-
-        const dimensionsOptions = dimensions.map(d => d.name);
+        const dimensionsOptions = {
+            Popular: popularDimensions.map(pD => pD.name),
```
Comment (tVeratti) [inline components/universalSearch/searchBar.js]: We might as well start the move to `label` instead of name.

**Comment by mathom** [comment]
[100% (0.0%) vs master 100%](https://jenkins.topbox.io/job/Topbox/job/rci-components/job/PR-622/1/)

File: components/universalSearch/searchBar.js (hunk 1 above)
Comment (troyblank) [reply components/universalSearch/searchBar.js]: Great call, forgot that changed. Updated!

**Comment by mathom** [comment]
[100% (0.0%) vs master 100%](https://jenkins.topbox.io/job/Topbox/job/rci-components/job/PR-622/2/)

**Comment by troyblank** [comment]
When in Rome.

**Pull request merged**